database/factories/BankIntegrationFactory.php

```diff
@@ -25,6 +26,7 @@ class BankIntegrationFactory extends Factory
     public function definition()
     {
         return [
+            'integration_type' => BankIntegration::INTEGRATION_TYPE_NONE,
```

Comment:
I think we can drop the none type completely, as we'll set the column to nullable in the migration.